README.md

```diff
@@ -1,2 +1,26 @@
-# ocds_communication_extension
+# Communication
+
 Adds a communication object to the tender to describe the modalities of communication about key events.
+
+## Legal context
+
+In the European Union, this extension's fields correspond to [eForms BT-124 and BT-127](https://github.com/eForms/eForms). See [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/master/en/) for the correspondences to Tenders Electronic Daily (TED).
+
+## Example
+
+```json
+{
+  "tender": {
+    "atypicalToolUrl": "https://ecomm-procurement.example.net",
+    "futureNoticeDate": "2020-06-17T00:00:00+01:00"
+  }
```

Comment:
Missing a `communication` object around the two fields.

Reply:
Done